inst/default_options.csv

```diff
@@ -1,40 +1,41 @@
+GMB,GMB,2,CY2024Q4,CY2025Q3,CY2013Q1,GMB2013DHS,,,TRUE,,TRUE,TRUE,TRUE,2024,12,2021,2024,2021,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,
+GNB,GNB,1,CY2024Q4,CY2025Q3,CY2010Q2,GNB2010MICS,,,TRUE,,TRUE,TRUE,TRUE,2024,12,2021,2024,2021,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,
+HTI,HTI,2,CY2024Q4,CY2025Q3,CY2020Q1,HTI2019PHIA,HTI2019PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,2020,2024,2020,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,CY2026Q3,CY2027Q3,1
+KEN,KEN,2,CY2024Q4,CY2025Q3,CY2018Q4,KEN2018PHIA,KEN2018PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,2018,2024,2018,2024,subnational,FALSE,250,1000,1000,TRUE,-3,TRUE,1,TRUE,1,TRUE,1,subnational,sex_age_group,subnational,sex_age_group,subnational,sex_age_group,subnational,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,2
+LBR,LBR,1,CY2024Q4,CY2025Q3,CY2013Q2,LBR2013DHS,,,FALSE,,TRUE,TRUE,TRUE,2024,12,2021,2024,2021,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,1
+LSO,LSO,1,CY2024Q4,CY2025Q3,CY2020Q1,LSO2020PHIA,LSO2020PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,2020,2024,2020,2024,national,TRUE,250,1000,1000,TRUE,-3,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,1
+MLI,MLI,1,CY2024Q4,CY2025Q3,CY2013Q1,MLI2012DHS,,,FALSE,,TRUE,TRUE,TRUE,2024,12,,2024,,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,1
+MOZ,MOZ,3,CY2024Q4,CY2025Q3,CY2021Q3,MOZ2021PHIA,MOZ2021PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,,2024,2021,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,TRUE,TRUE,CY2026Q3,CY2027Q3,3
+MWI,MWI,5,CY2024Q4,CY2025Q3,CY2020Q3,MWI2020PHIA,MWI2020PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,2020,2024,2020,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,3
```

Comment:
Should this be 4 for the area level?

Test data is coming through with
* Country - level 0
* Region - level 1
* Zone - level 2
* District - level 3
* District + Metro - level 4

Reply:
No, we don't want to do this. The actual Malawi data does have 6 levels. So 5 is right